gradle.properties

```diff
@@ -1,5 +1,5 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-Xmx1024m -XX:MaxPermSize=512M
-ideaVersion=IC-LATEST-EAP-SNAPSHOT
+ideaVersion=PC-LATEST-EAP-SNAPSHOT
```

Comment:
Is there a reason to make the plugin PyCharm-specific? 

While it can be assumed that the main audience for this plugin is using PyCharm, we do have a non-trivial (~1k) number of users running pure IDEA. ;(

Reply:
This change doesn't affect the IDEs that are compatible with PyCharm. The set of compatible IDEs is determined by `<depends>` tags in plugin.xml. I haven't added any new restrictions there, so the compatibility with IntelliJ should remain intact.

This change compiles the plugin in the context of PyCharm by default (you can change `ideaVersion` on CI if needed to test the build against several IDEs, but we will have to make `intellij.plugins` in build.gradle a conditional expression similar to [what I have in intellij-micropython](https://github.com/vlasovskikh/intellij-micropython/blob/8ca59f479f6b48714a22127d67fbf4ac5912d1a8/build.gradle#L23)).

I have changed `IC` to `PC` because I wanted to run the actual Python formatter in my tests and I needed the Python plugin for that. The plugin comes with PyCharm by default, therefore it was just easier to test it like that. In addition, it seemed nice to be able to run and debug code in the development environment in PyCharm by default, since it's the most popular IDE for this plugin.